pkgs/top-level/python-packages.nix

```diff
@@ -2466,6 +2466,8 @@ in {
 
   colored = callPackage ../development/python-modules/colored { };
 
+  xapian-bindings-python = callPackage ../development/libraries/xapian/bindings/python.nix { };
```

Comment:
it's IMHO sufficient to name this either `xapian-bindings` or even just `xapian`. The `python` suffix is not needed as it's part of the `pythonPackages` set.

Reply:
if there's no PyPI name the name best be the same as the original package, so `xapian`